src/Symfony/Component/Serializer/Tests/Normalizer/Features/CallbacksTestTrait.php

```diff
@@ -29,7 +29,7 @@ public function testCallbacks($callbacks, $valueBar, $result)
     }
 
     /**
-     * @dataProvider provideInvalidCallbacks()
+     * @dataProvider provideInvalidCallbacks
```

Comment:
By the way. I practised my git skills on this PR. 

I added this change while merging.

Reply:
Thanks